wagtailio/core/templates/core/blocks/brands_block.html

```diff
@@ -2,17 +2,26 @@
 
 {% image value.background original as bg_image %}
 
-{% if value.brands %}
+{% if value.brands2 %}
     <section class="logo-list">
 
         <div class="logo-list__container">
 
             <h2 class="logo-list__heading heading heading--secondary heading--top-border">{{ value.title }}</h2>
 
             <div class="logo-list__carousel carousel">
-                {% for brand in value.brands %}
-                    {% image brand width-300 as brand_image %}
-                    <div><img src="{{ brand_image.url }}" alt="{{ brand.title }}" /></div>
+                {% for brand in value.brands2 %}
+
+                    {% image brand.image width-300 as brand_image %}
+                    {% if brand.url %}
+                        <a href="{{ brand.url }}">
```

Comment:
If these are external links, should we add `target="_blank"`?

Reply:
I think we need that yes! 👍

Reply:
No, don't do it! https://github.com/wagtail/wagtail/issues/1167#issuecomment-199594073